build.ps1

```diff
@@ -3,6 +3,7 @@ param(
     [switch]$WinX64,
     [switch]$WinX86,
     [switch]$LinuxX64,
+    [switch]$MacOs,
     [switch]$Osx,
```

Comment:
shoudn't we remove this?

Reply:
I did not want to break somebody's scripts - it's just an alias.